poker-pals/src/components/Login.vue

```diff
@@ -104,8 +104,8 @@
       });
 
       // Navigate to tables page if authentication was successful
-      this.socket.on("authenticated", () => {
-        this.$router.push({ name: "Tables" });
+      this.socket.on("authenticated", userID => {
+        this.$router.push({ name: "Tables", params: {authenticated: true, userID: userID}});
```

Comment:
I did this for my changes too. That's how we route the important information between pages. 

I thought the focus of your PR was to only route information between Tables and Poker page. Since you already did this part, we can assign the ticket I had to you too. I'm not sure if my PR will provide any value if it's already done here

Reply:
i just needed something for the login to connect to the tables and then to the poker, not sure if its appropriate or if we will use this set up

Reply:
to clarify, in order to remove the hardcoded data i needed true login authentication, so i added these few lines of code to do the trick

Reply:
I think using params is the way to go. I spent way too long on trying to do it other ways through state management but this seems to be the easiest method

Reply:
Do you want to then share the ticket that is assigned to me? Cause you're lready pushing the changes

Reply:
I will move this conversation to the messenger.